app/vmagent/remotewrite/client.go

```diff
@@ -441,8 +439,9 @@ again:
 		c.blocksSent.Inc()
 		return true
 	}
+
 	metrics.GetOrCreateCounter(fmt.Sprintf(`vmagent_remotewrite_requests_total{url=%q, status_code="%d"}`, c.sanitizedURL, statusCode)).Inc()
-	if statusCode == 409 || statusCode == 400 {
+	if statusCode == http.StatusConflict || statusCode == http.StatusBadRequest {
```

Comment:
This is a hard question - what is better to use - numeric http status codes or `http.Status*` named constants from the `http` package. From my practice it is better to stick with numeric http status codes, since they are easier to deal with during debugging and refactoring. For example, I had to look into `go doc http.StatusConflict | grep Conflict` in order to verify whether it really resolves to `409`, while reviewing this change.

Currently VictoriaMetrics source code contains both numeric http status codes and `http.Status*` constants in different places. I think we need to stick to numeric http status codes everywhere for the sake of consistency, and document this decision somewhere. This is a task for the future.

Reply:
noted, I'll use a numeric status code from now on.